.github/workflows/eo-cd.yml

```diff
@@ -236,6 +236,7 @@ jobs:
           chart-folder: build/infrastructure/eo/chart
           chart-repository: ${{ github.repository_owner }}/helm-charts
           chart-version: ${{ needs.define_build.outputs.chart_version }}
+          version_yaml_path_prefix: ''
```

Comment:
What is this? ❓

Reply:
Very important 😎
Fixes the pipeline for preview.

Reply:
Honestly should probably have been a completely seperate PR.

Reply:
No no :) Its fine with me. Just happy to know why it's there.